### PR TITLE
Feat/password

### PR DESCRIPTION
**Pull request by cchigo**
This is the final UI change. Please review. I await your feedback or recommendation. 
PS: I really hope you like it :)
 Thank you

### CODE REVIEW

File: app/src/main/res/layout/activity_get_details.xml [hunk 1]
```diff
@@ -49,197 +65,260 @@
...
+                    android:textColor="@color/white"
+                    android:fontFamily="@font/avenir_bold"
+                    android:textAllCaps="false"
                     tools:layout_editor_absoluteX="117dp"
                     tools:layout_editor_absoluteY="107dp" />
 
             </LinearLayout>
 
-            <LinearLayout
-                android:id="@+id/details_layout"
```
Comment (akshaykalola28) [inline app/src/main/res/layout/activity_get_details.xml]: Here! You removed the details_layout Id which is related connected to java file. Please assign this id to the relevant layout.

Comment (akshaykalola28) [reply app/src/main/res/layout/activity_get_details.xml]: And also manage java object according to it.

Comment (cchigo) [reply app/src/main/res/layout/activity_get_details.xml]: Alright. changing it right away

Comment (cchigo) [reply app/src/main/res/layout/activity_get_details.xml]: updated

**Pull request merged**